rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -536,6 +554,112 @@ void saveOldFederation_after_RSKIP123_should_save_null() {
         );
     }
 
+    private static Stream<Arguments> provideFederationBtcUTXOsTestArguments() {
+        return Stream.of(
```

Comment:
Might be simpler to pass the activations object directly with ceratin RSKIPs already active

Reply:
Agree

Reply:
Updated.